kernel/src/main/java/org/kframework/backend/kore/ModuleToKORE.java

```diff
@@ -137,6 +147,9 @@ public String convert(boolean heatCoolEq) {
                     sb.append("hooked-");
                 }
             }
+            if (tokenSorts.contains(sort)) {
+                att = att.add(HAS_DOMAIN_VALUES, "");
```

Comment:
Don't think you need the second parameter here

Reply:
Fixed.